cmd/influxd/backup/backup.go

```diff
@@ -166,5 +323,26 @@ backup downloads a snapshot of a data node and saves it to disk.
         -host <host:port>
                           The host to connect to snapshot.
                           Defaults to 127.0.0.1:8088.
+        -database <name>
```

Comment:
Whatever works for you is fine with me. One way may be more maintainable
than the other, but it's not a blocker obviously.

On Wednesday, December 30, 2015, Paul Dix <notifications@github.com> wrote:

> In cmd/influxd/backup/backup.go
> <https://github.com/influxdb/influxdb/pull/5224#discussion_r48638035>:
>
> > @@ -166,5 +323,26 @@ backup downloads a snapshot of a data node and saves it to disk.
> >          -host <host:port>
> >                            The host to connect to snapshot.
> >                            Defaults to 127.0.0.1:8088.
> > +        -database <name>
>
> I was following what @benbjohnson <https://github.com/benbjohnson> had
> done in his. Maybe this was because of the PATH at the end that didn't
> have any flag associated with it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/5224/files#r48638035>.
>